content/en/docs/how-tos/adding-a-cluster-profile.md

```diff
@@ -165,3 +184,7 @@ exit_with_test &
 N.b. both containers start simultaneously, so the test may exit before the VPN
 client starts.
 {{< /alert >}}
+
+[openshift_release]: https://github.com/openshift/release.git
```

Comment:
:TIL
Cleaner way of defining links.

Reply:
Yes, it's a core part of Markdown: https://daringfireball.net/projects/markdown/syntax#link.

It makes satisfying my obsession with 80-column text much easier.